docs/src/pages/en/components/media-control-bar.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: <media-control-bar>
+description: Media Control Bar
+layout: ../../../layouts/ComponentLayout.astro
+source: https://github.com/muxinc/media-chrome/tree/main/src/js/media-control-bar.js
+---
+
+import SandpackContainer from "../../../components/SandpackContainer.astro";
+
+The `<media-control-bar>` container component is used to make layout and styling of your controls as a "control bar" easier.
+
+## Default usage
+
+By default, `<media-control-bar>` will try to scale down other Media Chrome components if there is not enough room and will grow to the width of the `<media-controller>`.
```

Comment:
This example is clearly not looking too great at smaller player sizes (probably intentional to show the impact of the scaling) – but rather than have it not look to great, maybe we should make a suggestion about further reading for styling the control bar with responsiveness in mind (and link out to our styling or responsive guide)